app/models/collection_draft.rb

```diff
@@ -94,8 +94,14 @@ def update_draft(params, editing_user_id)
 
       # Convert {'0' => {'id' => '123'}} to [{'id' => '123'}]
       params = convert_to_arrays(params.clone)
-      # Convert parameter keys to CamelCase for UMM
+
       json_params = params.to_camel_keys
+
+      # Convert parameter keys to CamelCase for UMM;
```

Comment:
I think you accidentally moved this comment also.

Reply:
yes. good catch, thanks.